.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 bin
 vendor
+composer.lock
```

Comment:
Interesting why you've put `composer.lock` in the `.gitignore` and committed it as well?